lib/saluki-components/src/destinations/prometheus/mod.rs

```diff
@@ -334,13 +334,11 @@ fn write_metrics(
             PrometheusValue::Histogram(histogram) => {
                 // Write the histogram buckets.
                 for (upper_bound_str, count) in histogram.buckets() {
-                    if count != 0 {
-                        write!(payload_buffer, "{}_bucket{{{}", &prom_context.metric_name, tags_buffer).unwrap();
-                        if !tags_buffer.is_empty() {
-                            payload_buffer.push(',');
-                        }
-                        writeln!(payload_buffer, "le=\"{}\"}} {}", upper_bound_str, count).unwrap();
+                    write!(payload_buffer, "{}_bucket{{{}", &prom_context.metric_name, tags_buffer).unwrap();
+                    if !tags_buffer.is_empty() {
+                        payload_buffer.push(',');
                     }
+                    writeln!(payload_buffer, "le=\"{}\"}} {}", upper_bound_str, count).unwrap();
```

Comment:
I had originally done the change to not show buckets with a zero count to clean up the output, but realized it just sort of hides the necessary data to show which buckets things actually fall between.